hack/functests.sh

```diff
@@ -25,10 +25,15 @@ DOCKER_TAG_ALT=${DOCKER_TAG_ALT:-devel_alt}
 source hack/common.sh
 source hack/config.sh
 
+_auto_detected_latest_release_tag=$(curl -s https://github.com/kubevirt/kubevirt/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
```

Comment:
I think we need to test upgrades against 2 releases: the last patch release of the same minor release, and of the last minor release. So e.g. atm master should test against v0.18.0 and v0.17.2. 
Also, I think we need to test if are running on a release branch, in order to not test downgrades. E.g. on the release-0.17 branch we don't want to test against the v0.18.0 release, but the latest patch releases of v0.17 and v0.16 releases (with the exception, that 0.16 to 0.17 upgrades are not supported, so should be skipped).

Reply:
@slintes I removed the auto detection logic for now. 

I think this is something we're going to have to manually keep up with. I can't think of a way to auto detect this correctly once we bring things like release branches into the picture. It doesn't appear trivial to determine what the previous release is.